src/System.Data.Common/tests/System/Data/Common/DataColumnMappingCollectionTest.cs

```diff
@@ -472,7 +472,7 @@ public void AddRange_Array_PassingNullThrowsException()
             Array array = null;
             DataColumnMappingCollection dataColumnMappingCollection = new DataColumnMappingCollection();
 
-            Assert.Throws<ArgumentNullException>(() => dataColumnMappingCollection.AddRange(array));
+            Assert.Throws<ArgumentNullException>("values", () => dataColumnMappingCollection.AddRange(array));
```

Comment:
@stephentoub  Thanks for the recommended way of testing with parameter names.  I have updated the tests per this recommendation.  If you have any questions or comments, please let me know.

Thanks again,

Michael